packages/core/utils/src/path.js

```diff
@@ -3,16 +3,13 @@
 import type {FilePath} from '@parcel/types';
 import path from 'path';
 
-const COMMON_SEPARATORS = ['/', '\\'];
+const SEPARATOR_REGEX = /[/\\]+/g;
 
-export function normalizeSeparators(filePath: FilePath): FilePath {
-  let ret = filePath;
-
-  for (let separator of COMMON_SEPARATORS) {
-    ret = ret.split(separator).join(path.sep);
-  }
-
-  return ret;
+export function normalizeSeparators(
+  filePath: FilePath,
+  replaceValue: string = path.sep,
+): FilePath {
+  return filePath.replace(SEPARATOR_REGEX, replaceValue);
```

Comment:
Regexes should be way quicker here... https://jsperf.com/normalising-windows-paths/1